testing/test_assertrewrite.py

```diff
@@ -115,18 +115,18 @@ def test_dont_rewrite(self) -> None:
         assert isinstance(m.body[1], ast.Assert)
         assert m.body[1].msg is None
 
-    def test_dont_rewrite_plugin(self, testdir):
+    def test_dont_rewrite_plugin(self, pytester: Pytester):
```

Comment:
It will be great (though not absolutely necessary) if, while adding the `: Pytester` annotation, you also add `-> None` annotations to tests. Without it, mypy (the type checker) still considers the typing to be incomplete. In this case it will be

```suggestion
    def test_dont_rewrite_plugin(self, pytester: Pytester) -> None:
```